server/github_token_handler.go

```diff
@@ -0,0 +1,47 @@
+package server
+
+import (
+	"net/http"
+
+	"github.com/mattermost/release-bot/client"
+	"github.com/mattermost/release-bot/store"
+	log "github.com/sirupsen/logrus"
+)
+
+type githubTokenHandler struct {
+	ClientManager     client.GithubClientManager
+	EventContextStore store.EventContextStore
+}
+
+func (gh *githubTokenHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	log.Info("New Token Request is Received!")
+	if r.Method != "GET" {
+		http.Error(w, "Only GET method is supported!", http.StatusMethodNotAllowed)
+		return
+	}
+	botToken := r.URL.Query().Get("token")
```

Comment:
instead of passing in URL is it possible to pass it as a POST payload?

Reply:
Will change it to the post and return JSON. That is great idea, so we may add extra fields in future.

Reply:
Endpoint method is changed to POST.